src/es/cms/typeahead.ts

```diff
@@ -12,21 +12,29 @@ const LABELS = {
 
 export async function getCmsSuggestions(query: string): Promise<TypeAheadSuggestion[]> {
   const types = [CmsType.Article, CmsType.Publication, CmsType.Collection]
-  const { results, totalCount } = await getCmsFromElasticSearch({ q: query, types })
 
-  const formattedResults = results.map(({ _source: result }) => {
-    const { title, field_short_title: shortTitle, type, uuid } = getValuesFromES(result) as any
+  return Promise.all(
```

Comment:
Shouldn't we use Promise.allSettled to avoid having nothing results when one promise fails?

Reply:
Yeah, good idea! I took it a bit further and refactored the code so any one of the searches can fail and as long as one succeeds it will still respond with those.